modules/sitemaps/sitemap-stylist.php

```diff
@@ -0,0 +1,620 @@
+<?php
+/**
+ * The XSL used to style sitemaps is essentially a bunch of
+ * static strings. This class handles the construction of
+ * those strings.
+ *
+ * @package Jetpack
+ * @since 4.5.0
+ */
+
+/**
+ * Builds the XSL files required by Jetpack_Sitemap_Manager.
+ *
+ * @since 4.5.0
+ */
+class Jetpack_Sitemap_Stylist {
+
+	/**
+	 * Convert named entities, strip all HTML except anchor tags,
+	 * and interpolate with vsprintf. This is a helper function
+	 * for all the internationalized UI strings in this class
+	 * which have to include URLs.
+	 *
+	 * Note that $url_array should be indexed by integers like so:
+	 *
+	 * array(
+	 *   1 => 'example.com',
+	 *   2 => 'example.org',
+	 * );
+	 *
+	 * Then '%1$s' in the format string will substitute 'example.com'
+	 * and '%2$s' will substitute 'example.org'.
+	 *
+	 * @access private
+	 * @since 4.5.0
+	 * @link http://php.net/manual/en/function.vsprintf.php Format string documentation.
+	 *
+	 * @param string $format A vsprintf-style format string to be sanitized.
+	 * @param array  $url_array The string substitution array to be passed to vsprintf.
+	 *
+	 * @return string The sanitized string.
+	 */
+	private static function sanitize_with_links( $format, $url_array ) {
+		return vsprintf(
+			wp_kses(
+				ent2ncr( $format ),
+				array(
+					'a' => array(
+						'href'  => true,
+						'title' => true,
+					),
+				)
+			),
+			$url_array
+		);
+	}
+
+	/**
+	 * Returns the xsl of a sitemap xml file as a string.
+	 *
+	 * @access public
+	 * @since 4.5.0
+	 *
+	 * @return string The contents of the xsl file.
+	 */
+	public static function sitemap_xsl() {
+		$title          = esc_html( ent2ncr( __( 'XML Sitemap', 'jetpack' ) ) );
+		$header_url     = esc_html( ent2ncr( __( 'URL', 'jetpack' ) ) );
+		$header_lastmod = esc_html( ent2ncr( __( 'Last Modified', 'jetpack' ) ) );
+
+		$description = Jetpack_Sitemap_Stylist::sanitize_with_links(
+			__(
+				'This is an XML Sitemap generated by <a href="%1$s" target="_blank">Jetpack</a>, meant to be consumed by search engines like <a href="%2$s" target="_blank">Google</a> or <a href="%3$s" target="_blank">Bing</a>.',
+				'jetpack'
+			),
+			array(
+				1 => 'http://jetpack.com/',
+				2 => 'https://www.google.com/',
+				3 => 'https://www.bing.com/',
+			)
+		);
+
+		$more_info = Jetpack_Sitemap_Stylist::sanitize_with_links(
+			__(
+				'You can find more information on XML sitemaps at <a href="%1$s" target="_blank">sitemaps.org</a>',
+				'jetpack'
+			),
+			array(
+				1 => 'http://sitemaps.org',
+			)
+		);
+
+		$generated_by = Jetpack_Sitemap_Stylist::sanitize_with_links(
+			__(
+				'<em>Generated</em> by <a href="%s" target="_blank">Jetpack for WordPress</a>',
```

Comment:
`Jetpack_Sitemap_Stylist::sanitize_with_links` strips out the `em` tag. Just remove the `em`.